creator-node/scripts/delistContent/index.js

```diff
@@ -318,10 +318,9 @@ async function checkIsCIDDelisted (cid) {
   return { type: 'CID', value: cid, delisted: false }
 }
 
-async function checkIsTrackDelisted (id) {
+async function checkIsTrackDelisted (cid) {
   try {
-    const encodedId = hashIds.encode(id)
-    await axios.head(`${CREATOR_NODE_ENDPOINT}/tracks/stream/${encodedId}`)
+    await axios.head(`${CREATOR_NODE_ENDPOINT}/tracks/cidstream/${cid}`)
```

Comment:
need to test that this actually works, fyi @theoilie maybe we can sync quickly on this Monday

Reply:
I don't think this would work without the signature but it's fine to just remove the verification here. Nodes will be polling delist status rather than having trusted notifier push the status